toasty/image.py

```diff
@@ -419,24 +438,34 @@ class Image(object):
     have "bitmap" RGB(A) images and "science" floating-point images.
 
     """
-    _mode = None
     _pil = None
     _array = None
+    _mode = None
+    _default_format = 'png'
+    _wcs = None
 
     @classmethod
-    def from_pil(cls, pil_img):
+    def from_pil(cls, pil_img, wcs=None, default_format=None):
         """
         Create a new Image from a PIL image.
 
         Parameters
         ----------
         pil_img : :class:`PIL.Image.Image`
             The source image.
+        wcs : :class:`~astropy.wcs.WCS`
+            The WCS coordinate system for the image.
```

Comment:
This doc text should mention that the WCS is optional.

Reply:
Fixed